### DISCUSSION TITLE
帮帮看看这个驱动盘可以用吗

### DISCUSSION THREAD

**Discussion by annabbwww** [share121/inter-knot#2315] [其他]
![Screenshot_20240816_013120_com miHoYo Nap](https://github.com/user-attachments/assets/4d0fe328-2589-4acf-bf65-4644b2fc4ed7)
莫名其妙的发现背包里多了一个双爆词条的驱动盘！
已经给小可琳用了！

**Comment by FYGT27** [comment]
我的以太双爆驱动盘可是全点大防御去了，我只能说有点怕痛😡
![Screenshot_2024-08-15-14-39-40-009_com miHoYo Nap](https://github.com/user-attachments/assets/720c91f5-6bb1-4740-9b0d-be8721318f2b)
